amundsen_application/api/dashboard/v0.py

```diff
@@ -1,11 +1,12 @@
 import logging
```

Comment:
nit: Are you opposed to moving all of this to `api/metadata/v0`? This is still a call to the metadata service, and all other dashboard support for the MVP will still be calling `api/metadata/v0` since the code can be reused

Reply:
I can move it back to `metadata`, though I would like to eventually revisit this file. It is becoming a monolithic file with the majority of our APIs here. For example we could move all Bookmark APIs into a separate file, and this would support fetching, adding, and removing bookmarks for any resource type, dashboard, table, etc.

Reply:
If and when things are revisited let's also not forget about https://github.com/lyft/amundsen/issues/307